c++/include/orc/Writer.hh

```diff
@@ -262,6 +262,17 @@ namespace orc {
      * @return if not set, the default is false
      */
     bool getUseTightNumericVector() const;
+
+     /**
+     * Set the buffer capacity of output stream.
+     */
+    WriterOptions& setOutputBufferCapacity(uint64_t capacity);
```

Comment:
Could you describe the characteristic of this capacity, immutable or updatabe?

Reply:
To be clear, I guess you can say like `initial capacity` and mention when and how this will be changed from the initial value.

Reply:
IMO, the user may not know what is the `OutputBuffer` or how many buffers will be allocated. We need to add comment to cover some basics.

Reply:
Add some descriptions to the API.